apps/app/src/configs/fulfillment.boson.ts

```diff
@@ -0,0 +1,23 @@
+export type FulfillmentSettings = {
```

Comment:
Could we put this elsewhere? a little weird that fulfillment.neutron.ts has to import from fulfillment.boson.ts

Reply:
yep